Vulkan-Pipeline-Assistant/spvstructwidget.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SPVSTRUCTWIDGET_H
+#define SPVSTRUCTWIDGET_H
+
+#include "spvwidget.h"
+
+class QPushButton;
```

Comment:
For clarity maybe add a space between forward declarations and implementations.

Reply:
I'll add a note in trello for refactoring to check over style and ensure everything is up to par, including a note of this in its comments.